app/src/main/java/at/irfc/app/util/CachedRemoteResource.kt

```diff
@@ -30,10 +31,13 @@ inline fun <EntityType : Any, ApiType : Any> cachedRemoteResource(
     val flow = if (shouldFetch(cachedData)) {
         emit(Resource.Loading(cachedData))
 
+        @Suppress("TooGenericExceptionCaught")
         try {
             update(fetch())
             query().map { Resource.Success(it) }
         } catch (tr: Throwable) {
+            // TODO provide a better way to handle different errors
```

Comment:
Todo will be checked in as well?

Reply:
Yes, it's a TODO for later, not sure what the best approach / the real needs are here. Fine for now – will figure that out with the first features, I guess. :)